llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -592,8 +662,223 @@ static Error applyAddends(MCCASReader &Reader,
   return Error::success();
 }
 
-Expected<uint64_t> DebugInfoSectionRef::materialize(MCCASReader &Reader,
-                                                    raw_ostream *Stream) const {
+static Expected<uint64_t> getFormSize(dwarf::Form FormVal, dwarf::FormParams FP,
```

Comment:
when upstreaming we should sink this function into libDebugInfo and unittest it there